swupd/packs.go

```diff
@@ -314,13 +349,13 @@ func copyFromDelta(tw *tar.Writer, delta *Delta) (fallback bool, err error) {
 		return true, err
 	}
 	if !fi.Mode().IsRegular() {
-		return true, fmt.Errorf("delta %s is not a regular file", delta.Path)
+		return true, fmt.Errorf("file %s is not a regular file", path)
 	}
 	hdr, err := getHeaderFromFileInfo(fi)
 	if err != nil {
 		return true, err
 	}
-	hdr.Name = "delta/" + fi.Name()
+	hdr.Name = pathPrefix + fi.Name()
```

Comment:
`filepath.Join` here and you don't have to append `/` everywhere.

Reply:
nice catch